packages/paymaster/src/interfaces/IPaymaster.ts

```diff
@@ -1,5 +1,7 @@
 import { UserOperation } from '@biconomy/core-types'
 
+
 export interface IPaymaster {
```

Comment:
Adding extra parameter in the method will not lead to function overriding. Instead we can use generic type to enable this 

```
export interface IPaymaster<T> {
  getPaymasterAndData(userOp: Partial<UserOperation>, paymasterServiceData?: T): Promise<string>;
}
```

Reply:
I can define this in IHybridPaymaster - any interface that extends this interface